.github/ISSUE_TEMPLATE/rule_change.yml

```
@@ -0,0 +1,56 @@
+name: Rule change
+description: As rule could be improved or extended?
+title: "`rule-name` should do xyz"
+labels:
+  - enhacement
```

```suggestion
  - enhancement
```

Alternatively, it could use a `change request` label like we have in Refined GitHub

---

`improvement` is a synonym of `enhancement`

The choice would be between "change" and "improve". Technically every change is an improvement, but in this case I'm seeing "change" as `I don't agree with this rule, please alter it`, like

- https://github.com/sindresorhus/eslint-plugin-unicorn/issues/2266

---

Let's go with "change request" then. Doesn't really matter that much.

---

```suggestion
  - change request
```